packages/hydra-cli/src/templates/variants/variants.mst

```diff
@@ -53,6 +56,16 @@ export class {{name}} {
     {{camelName}}{{^required}}?{{/required}}{{#required}}!{{/required}}: typeof {{tsType}};
     {{/is.union}}
 
+    {{#is.entity}}
+        @Field(type => {{tsType}}, {
+            {{^required}}nullable: true,{{/required}}
+            {{#description}}description: `{{{description}}}`{{/description}} })
+        {{camelName}}{{^required}}?{{/required}}{{#required}}!{{/required}}:{{tsType}};
+
+        @IntField({nullable: true})
```

Comment:
The field decorator is different than expected(`@StringField`). I suggest adding `dbOnly: true` option to the decorated function so the field will not be visible in the graphql API and also a default description i.e `This field is required to make the relationship possible and it is the id of the related entity`.